tools/pythonpkg/duckdb_python.cpp

```diff
@@ -103,6 +103,8 @@ static void InitializeConnectionMethods(py::module_ &m) {
 	         py::arg("connection") = py::none())
 	    .def("rollback", &PyConnectionWrapper::Rollback, "Roll back changes performed within a transaction",
 	         py::arg("connection") = py::none())
+	    .def("read_csv", &PyConnectionWrapper::ReadCSV, "Read the CSV file identifier by 'name'", py::arg("name"),
+	         py::arg("connection") = py::none(), py::kw_only {})
```

Comment:
If you're going to use `kwargs` here, you should list all the supported arguments in the docstring (preferably using the RestructedText Sphinx format)

Reply:
Some reference docs for the syntax: https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

Reply:
Ah nice, found how to do it with pybind11https://pybind11.readthedocs.io/en/stable/advanced/misc.html#generating-documentation-using-sphinx, will add it :)